packages/commerce-sdk-react/src/hooks/ShopperBaskets/mutation.test.ts

```diff
@@ -75,12 +74,38 @@ const deletedCustomerBaskets: BasketsResult = {
 }
 
 // --- TEST CASES --- //
-/** All Shopper Baskets mutations except `deleteBasket` have the same cache update logic. */
-type NonDeleteMutation = Exclude<ShopperBasketsMutation, 'deleteBasket'>
+/** All Shopper Baskets mutations except these have the same cache update logic. */
+type EmptyResponseMutations = Exclude<
```

Comment:
I think this is backwards, `deleteBasket` etc. are empty and the rest return `Basket`.

Reply:
Correct. I changed the type to `NonEmptyResposeMutations`.